neuralpp/symbolic/sympy_expression.py

```diff
@@ -196,6 +196,8 @@ def symbolic_integral(
                 a = big_f.replace(index.sympy_object, lower_bound.sympy_object)
             with profiler.profile_section("compute diff"):
                 diff = b - a
+                if all([gen.is_number for gen in diff.gens]):
+                    diff = diff.expr
```

Comment:
If all generators are numbers, we want to substitute the polynomial with a numerical value